# ==============================================================================
# REGRESSOR COMMIT DIFF
# ==============================================================================
# File: netwerk/protocol/http/PHttpBackgroundChannel.ipdl
# Commit: 19dcca77a1cf
# Full Hash: 19dcca77a1cf1c590dfda293418aec84cce701de
# Author: Junior Hsu <juhsu@mozilla.com>
# Date: 2020-06-24 09:31:07
# Regressor Bug: 1633935
# File Overlap Count: 1
# Description:
#   Bug 1633935 - P1 allow OnStartRequest go thru pBg IPC, r=mayhemer
#   
#   Differential Revision: https://phabricator.services.mozilla.com/D73529
# ==============================================================================

diff -r 91c26c5b1e33 -r 19dcca77a1cf netwerk/protocol/http/PHttpBackgroundChannel.ipdl
--- a/netwerk/protocol/http/PHttpBackgroundChannel.ipdl	Tue Jun 23 18:34:42 2020 +0000
+++ b/netwerk/protocol/http/PHttpBackgroundChannel.ipdl	Tue Jun 23 16:56:15 2020 +0000
@@ -26,6 +26,11 @@
   // is processed. For synchronizing the event sequence.
   async OnStartRequestSent();
 
+  async OnStartRequest(nsHttpResponseHead  responseHead,
+                       bool                useResponseHead,
+                       nsHttpHeaderArray   requestHeaders,
+                       HttpChannelOnStartRequestArgs args);
+
   // Combines a single OnDataAvailable and its associated OnProgress &
   // OnStatus calls into one IPDL message
   // This is duplicated on PHttpChannel, which gets used for multi-part
